include/cantera/transport/GasTransport.h

```diff
@@ -386,6 +412,10 @@ class GasTransport : public Transport
      * (i,j).
      */
     std::vector<vector_fp> m_omega22_poly;
+    
+    //! Flag to indicate for which (i,j) interaction pairs the 
+    //! actual temperature is used instead of the reduced temperature
+    std::vector<vector_int> m_star_poly_actualT;
```

Comment:
This is only `true` or `false`, correct? While it may not be possible to use `std::vector<bool>` (which sometimes behaves funny), clarifying this as `m_star_poly_uses_actualT` may improve code readability.

Reply:
I renamed the variable as suggested.